build_debian.sh

```diff
@@ -372,6 +372,10 @@ allow-hotplug eth0
 iface eth0 inet dhcp
 EOF
 
+# Disable acpi power saving mode used by some CPUs
+sudo LANG=C chroot $FILESYSTEM_ROOT /bin/bash -c 'echo "HandleLidSwitch=ignore" >> /etc/systemd/logind.conf'
```

Comment:
I think you only need this. IdleAction is ignore by default.

https://www.freedesktop.org/software/systemd/man/logind.conf.html

Reply:
I had checked-in the change to support your comment:
This change removed "IdleAction=ignore" since I verified it is ignored by default, so I agree that there is no point to add this option.
Would-you agree now to have this feature approved?